go/cli/mcap/cmd/info.go

```diff
@@ -62,7 +64,13 @@ func printInfo(w io.Writer, info *mcap.Info) error {
 		chunkCount := len(info.ChunkIndexes)
 		for k, v := range compressionFormatStats {
 			compressionRatio := 100 * (1 - float64(v.compressedSize)/float64(v.uncompressedSize))
-			fmt.Fprintf(buf, "\t%s: [%d/%d chunks] (%.2f%%) \n", k, v.count, chunkCount, compressionRatio)
+			fmt.Fprintf(buf, "\t%s: [%d/%d chunks] ", k, v.count, chunkCount)
+			fmt.Fprintf(buf, "[%.2d MB/%.2d MB (%.2f%%)] ",
+				(v.uncompressedSize / (1024 * 1024)), (v.compressedSize / (1024 * 1024)), compressionRatio)
+			if durationInSeconds > 0 {
+				fmt.Fprintf(buf, "[%.2f MB/sec] ", float64(v.compressedSize/(1024*1024))/durationInSeconds)
```

Comment:
This might produce funny results if you have a recording that uses zstd for the first half, then switches to lz4 for the remaining chunks (as an example). Oh well 🤷🏼